test/any/FetchWebIDLFile-test.js

```diff
@@ -0,0 +1,156 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+'use strict';
+
+describe('WebIDLFile', function() {
+  var IDLFile;
+  var GithubIDLFile;
+  var GitilesIDLFile;
+  var basePayload = 'Some sort of file content';
+  var githubPayload = 'Some sort of other content';
+
+  beforeAll(function() {
+    IDLFile = foam.lookup('org.chromium.webidl.IDLFile');
+    GithubIDLFile = foam.lookup('org.chromium.webidl.GithubIDLFile');
+    GitilesIDLFile = foam.lookup('org.chromium.webidl.GitilesIDLFile');
+  });
+
+  beforeEach(function() {
+    foam.CLASS({
+      package: 'org.chromium.webidl.test',
+      name: 'MockHTTPRequest',
+      extends: 'foam.net.HTTPRequest',
+
+      constants: {
+        SIMPLE_TEST_URL: 'http://test.url/someFile.idl',
+        GITHUB_TEST_URL: 'https://github.com/mozilla/gecko-dev/raw/0/someFile.idl',
+        GITILES_TEST_URL: 'https://chromium.googlesource.com/chromium/src/+/0/someFile.idl?format=TEXT'
+      },
+
+      properties: [
+        {
+          name: 'urlMap',
+          factory: function() {
+            var base64Potato = 'UG90YXRv';
```

Comment:
Also, we should add a comment that this is the text `Potato` in base64 encoding.